pages/learn/welcome/production-plan.md

```diff
@@ -75,6 +75,8 @@ After your application development is mostly complete, it's important to do some
 
 At this point it makes sense to move from development {{ $names.os.lower }} images to full production images, ensuring that your application still runs properly without full local console access. You can still [access][ssh] the host OS and application containers via the {{ $names.company.lower }} dashboard and CLI, but the production {{ $names.os.lower }} image will close all open inbound ports.
 
+For Production and Enterprise plan users with supported devices who require a stable, supported release, you should choose an [ESR version][esr] of the host OS. An ESR version guarantees that you will only have to update the host OS on the devices at most twice a year to ensure you are on a supported version.
```

Comment:
```suggestion
For Production and Enterprise plan users who require a stable, supported release, you should choose an [ESR version][esr] of the host OS. An ESR version guarantees that you will only have to update the host OS on the devices at most twice a year to ensure you are on a supported version.
```
And add another sentence that says that currently not all device types have ESR versions. The reasoning is, eventually we want all devices to have an ESR release, and as a first sentence it would be good to just introduce what ESR is, without the detail that some devices just don't have support yet.

Reply:
Resolved in https://github.com/balena-io/docs/pull/1240/commits/0c862efed33e7c00c53cde40afac327743934379